cvxpy/tests/test_KKT.py

```diff
@@ -0,0 +1,216 @@
+import cvxpy as cp
+from cvxpy.tests import solver_test_helpers as STH
+from cvxpy.tests.base_test import BaseTest
+
+
+class StandardTestLPs(BaseTest):
+
+    def test_lp_1(self):
+        # typical LP
+        sth = STH.lp_1()
+        if 'MOSEK' in cp.installed_solvers():
+            sth.solve(solver='MOSEK')
+        else:
+            sth.solve(solver='ECOS')
```

Comment:
Do all tests work with Mosek and Ecos? If so, why do we need this switch?

Reply:
Oh, I just included this switch in case a user ran these tests locally and didn’t have `MOSEK` installed. They do indeed pass with both choices.

Reply:
But if they pass with Mosek and Ecos, why include Mosek here at all?
Do you want to test mosek and Ecos, in case a user has both? You could parametrize the tests to do so.

Reply:
I explicitly included `MOSEK` because I was testing it out on my system XD. I will parametrize the tests in the final version of the PR!